run/jobs/main.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"math/rand"
+	"os"
+	"strconv"
+	"time"
+)
+
+func getEnvVars() (string, string, string, string) {
+	// Retrieve Job-defined env vars
+	var TASK_NUM = os.Getenv("TASK_NUM")
+	var ATTEMPT_NUM = os.Getenv("ATTEMPT_NUM")
+
+	// Retrieve User-defined env vars
+	var SLEEP_MS = os.Getenv("SLEEP_MS")
+	var FAIL_RATE = os.Getenv("FAIL_RATE")
+	return TASK_NUM, ATTEMPT_NUM, SLEEP_MS, FAIL_RATE
+}
+
+func main() {
+	TASK_NUM, ATTEMPT_NUM, SLEEP_MS, FAIL_RATE := getEnvVars()
+	log.Printf("Starting Task #%s, Attempt #%s ...", TASK_NUM, ATTEMPT_NUM)
```

Comment:
These log statements seem like a strong candidate to be deprecated by product improvements. Really like that we're making sure they happen! Maybe we should confirm if they'll be needed for awhile?